tests/stune/smoke_test_ramp.py

```diff
@@ -95,9 +95,15 @@ def test_boosted_utilization_signal(self):
             analyzer = Analyzer(ftrace, analyzer_const,
                                 window=(after_first_period, None),
                                 filters={"comm": rta_task_name})
-            statement = "(((SCHED_LOAD_SCALE - sched_boost_task:util) * BOOST) // 100) == sched_boost_task:margin"
-            error_msg = "task was not boosted to the expected margin: {:.2f}"\
-                        .format(boost / 100.)
-            self.assertTrue(analyzer.assertStatement(statement), msg=error_msg)
+            if boost >= 0:
+                statement = "(((SCHED_LOAD_SCALE - sched_boost_task:util) * BOOST) // 100) == sched_boost_task:margin"
```

Comment:
How about keeping only `statement = "..."` in the if/else block, and moving `error_msg = "..."` & `self.assertTrue(...)` outside of it ?